truss/templates/triton/proxy.conf

```diff
@@ -0,0 +1,29 @@
+
+server {
+    listen 8080;
+
+    # Liveness
+    location / {
+        add_header Content-Type text/plain;
+        return 200 "";
+    }
+
+    # Readiness
+    location ~ ^/v1/models/model$ {
+        proxy_redirect off;
+
+        rewrite ^/v1/models/model$ /v2/health/ready break;
```

Comment:
I'm confused. Doesn't this depend on the model id (1 in the case above)?

Reply:
1 is the model version. `model` is the name of the model we deploy (via the directory name)